rskj-core/src/main/java/org/ethereum/core/Block.java

```diff
@@ -182,6 +185,26 @@ public Block(byte[] parentHash, byte[] unclesHash, byte[] coinbase, byte[] logsB
         this.parsed = true;
     }
 
+    public void seal() {
+        this.sealed = true;
+        this.header.seal();
+    }
+
+    public boolean isSealed() {
+        return this.sealed;
+    }
+
+    // Clone this block allowing modifications
+    public Block cloneBlock() {
+        Block clone = new Block(this.getEncoded());
+        clone.sealed = false;
+        clone.parseRLP();;
+        // Allow modifications to header
+        clone.header.unseal();
```

Comment:
The header is a new one, created from RLP. It is only referenced by this clone block